docs/tech-reference/rpm.rst

```diff
@@ -0,0 +1,452 @@
+The Origin of RPM Content
+=========================
+
+Overview
+--------
+
+Pulp supports importing and publishing RPM content, but where does all this
+content come from? Who is generating it, and with what tools? The answer to
+these questions are answered in this document. Note that this document
+only covers Fedora and Red Hat Enterprise Linux. It does not cover SLES,
+OpenSuse, Scientific Linux, CentOS, Oracle Linux, or any other RPM-based
+distributions (yet).
+
+
+The RPM
+-------
+
+RPM is a packaging format. Many Linux distributions use RPM to distribute their
+software packages, although more do not. An RPM is a collection of metadata
+about the package as well as a payload. What the payload contains depends on the
+type of RPM. There are three different types of RPM:
+
+ * Source code RPMs (referred to as "SRPMs", ".src.rpm" file extension)
+
+ * Binary RPMs (referred to as "RPMs", ".rpm" file extension)
+
+ * Binary delta RPMs (referred to as "DRPMs", ".drpm" file extension)
+
+
+File Format
+^^^^^^^^^^^
+
+The RPM file format consists of a "lead" which identifies the file as an RPM, a
+"signature" section which can be used to ensure the integrity and authenticity
+of the RPM (via GPG), a "header" section which contains metadata about the
+packaged software (name, version, architecture, file list, etc.), and a
+"payload" which is a file archive (usually in the cpio format) compressed with
+gzip.
+
+SRPM
+^^^^
+
+The payload of a source RPM is simply a compressed tarball of source code and a
+file, called a spec file, that describes how to turn the source RPM into a
```

Comment:
Hmm. I think this comma usage is okay. In this case, dropping the portion of the sentence encased in the commas is still valid and reasonable, but the ``called a spec file`` further clarifies the sentence.

spec files do include scriptlets, but I didn't really want to dig that deeply into spec files. If you think it would be helpful I can make a note of it, though.